### ISSUE TITLE
Enable keyboard navigation by default

### ISSUE THREAD

**Issue by zmilonas** [subdavis/Tusk#167]
## This issue is a 

- [x] feature

Set the keyboard navigation setting to on by default.

**Comment by subdavis** [comment]
Agreed!  I want to do this after #151 because otherwise the yellow highlighting will confuse users.
